src/_includes/hero.njk

```diff
@@ -1,2 +1,7 @@
 <h1>{{ hero.title | safe }}</h1>
-<a class="btn btn--cta" href="#">Try Teacher Moments</a>
+<div class="hero-container">
+  <div class="step-by-step">
+    <img src="/static/images/step-by-step.png" alt="Step-by-step of process"/>
+    <a class="btn btn--cta" href="#">Try Teacher Moments</a>
+  </div>
+<div>
```

Comment:
This is just a style guide thing, but be sure to have an empty line at the end of every file. You'll probably be able to set it up in your editor to default to that on save :)